iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionContextsProvider.h

```diff
@@ -173,6 +173,17 @@ struct IECExpressionsCacheProvider
     ECExpressionsCache& Get(Utf8CP rulesetId) {return _Get(rulesetId);}
 };
 
+//===================================================================================
+// @bsiclass
+//===================================================================================
+enum class ECExpressionEvaluationResult

```

Comment:
I'd probably rename this from "result" to "status".

Reply:
Renamed